TROUBLESHOOTING.md

```diff
@@ -0,0 +1,70 @@
+Troubleshooting the plugin
+==============
+
+### Verifying the setup
+
+**0.** ensure the latest compatible version of IntelliJ IDEA Community Edition is installed (**2020.2.** or later).
+
+**1.** update plugin to the latest version (you can check the latest version on [Plugin's official page](https://plugins.jetbrains.com/plugin/13634-a-courses/versions)), but at least **1.4**.
+
+**2.** create a new project: [**File -> New -> Project...** and then **Empty Project**](https://plus.cs.aalto.fi/o1/2020/w01/ch02/#launch-intellij).
+
+**3.** Turn project into an A+ Project: [**A+ -> Turn Project Into A+ Project**](https://plus.cs.aalto.fi/o1/2020/w01/ch02/#the-a-courses-plugin) (and choose the rest of the settings asked for in the pop-up dialog).
+
+**4.** check the IDE for not having errors (you **should not** see the blinking red icon as shown). If there are errors, please share the "stacktrace" text with the course staff.
+![libraries](images/error_.png)
+
+**5.** review the project structure for correctness: **File -> Project Structure ->** ...
+
+**Project**
+
+![project](images/ts_project.png)
+
+**Modules**
+![modules 3](images/ts_modules_3.png)
+![modules 1](images/ts_modules_1.png)
+![modules 2](images/ts_modules_2.png)
+
+**Libraries (Scala SDK)**
+![libraries](images/ts_libs.png)
+
+**6.** download a **GoodStuff** module and run it by right-clicking and choosing **Run**
```

Comment:
Nit: "download a" -> download the"

Reply:
I see you take "proof-reading" seriously, thanks, fixed.